code/game/objects/machinery/doors/airlock.dm

```diff
@@ -34,6 +34,8 @@
 	var/secured_wires = 0	//for mapping use
 	var/no_panel = 0 //the airlock has no panel that can be screwdrivered open
 	var/emergency = FALSE
+	///used to determine various abandoned door effects
+	var/abandoned = FALSE
```

Comment:
varedit?

Reply:
Besides logically checking if we're already abandoned, in the future it altered in certain door subtypes to do the abandoned effect without the helper. I have no intention of allowing varedits of it.